public/components/_action-buttons.jade

```diff
@@ -1,3 +1,6 @@
 div.jsla-action-buttons
-    a.pure-button.jsla-button-xlarge( href="#{titoUrl}" itemprop="tickets" ) <i class="icon-ticket"></i> RSVP for event
+    if eventUrl
+        a.pure-button.jsla-button-xlarge( href="#{eventUrl}" itemprop="tickets" ) <i class="icon-ticket"></i> RSVP for event
```

Comment:
maybe we should use an explicit flag for the "no event" state? then we wouldn't have to worry about deleting out the old data and could have a simple "flip the flag" PR setup or automated

Reply:
or is this just so we can post the event info before the ticket page is up?

Reply:
The only piece of data we need to unset is the `datetime`. It flips everything since there is an overwrite in the `_hero-empty` component. I originally had a empty flag but figured unsetting to null would do the same thing. Might be good tho to just ignore the data so we don't have to "remake" it all.

Reply:
yeah, I like just setting `datetime` to `null` and ignoring any other "unrelated" data that is in the data json